.github/workflows/android_build.yml

```diff
@@ -0,0 +1,24 @@
+name: Android Build
+
+on: pull_request
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: set up JDK 11
+      uses: actions/setup-java@v3
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+        cache: gradle
+
+    - name: Execute permission for gradlew
+      run: chmod +x gradlew
+      
+    - name: Build 
+      run: ./gradlew build
```

Comment:
```euphony-collector-hub``` has androidTest, so after build, it would be good to run our tests.
Our app supports api level from 26 to 32, so how about executing test code at both api levels? 😸 

I think you can refer this link:
https://github.com/vinaygaba/Learn-Jetpack-Compose-By-Example/blob/master/.github/workflows/android.yml